src/test/java/space/obminyashka/items_exchange/rest/UserControllerIntegrationTest.java

```diff
@@ -152,13 +160,18 @@ private static Stream<Arguments> listInvalidEmail() {
     @Test
     @WithMockUser
     void resetUserPassword_whenPasswordResetSuccessfully_shouldSendMessage() throws Exception {
+        when(userService.findByUsernameOrEmail(any())).thenReturn(Optional.of(new User()));
+        when(userService.existsByEmail(any())).thenReturn(true);
```

Comment:
Do we still have both this mocks to invoke?

Reply:
corrected